crypto/CMakeLists.txt

```diff
@@ -195,6 +199,22 @@ if (OPENSSL_ASM OR OPENSSL_UPLINK)
             set(OPENSSL_PERLASM_SCHEME "linux64")
         elseif (APPLE)
             set(OPENSSL_PERLASM_SCHEME "ios64")
+        elseif (WIN32)
+            set(OPENSSL_ASM "OFF")
```

Comment:
This should be modified in CMakeLists.txt in root directory.

Reply:
Yeah I dont think this is working as passing the value higher was the key

Reply:
Also in root directory CMakeLists.txt there is logic setting `OPENSSL_BUILD_PLATFORM`
In Windows on ARM64 case I suggest it be set to `VC-WIN64-ARM-DESKTOP`
Not using the original `VC-WIN64-ARM` since it targets OneCore which is currently not supported by CMake.